static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
Am I right in understanding that `profileCopy` is an object we update locally (on the client), and then we only worry about persisting it to the server when we actually save? (`dispatch(saveProfile(...))`)